vm/string_test.go

```diff
@@ -39,34 +34,156 @@ func TestEvalInfixStringExpression(t *testing.T) {
 		{`"1234" <=> "1234"`, 0},
 		{`"1234" <=> "4"`, -1},
 		{`"abcdef" <=> "abcde"`, 1},
+	}
+
+	for _, tt := range tests {
+		evaluated := testEval(t, tt.input)
+		checkExpected(t, evaluated, tt.expected)
+	}
+}
+
+func TestStringOperation(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected interface{}
+	}{
+		{`"Stan " + "Lo"`, "Stan Lo"},
+		{`"Dog" + "&" + "Cat"`, "Dog&Cat"},
+		{`"Three " * 3`, "Three Three Three "},
+		{`"Zero" * 0`, ""},
+		{`"Minus" * 1`, "Minus"},
+	}
+
+	for _, tt := range tests {
+		evaluated := testEval(t, tt.input)
+		checkExpected(t, evaluated, tt.expected)
+	}
+}
+
+func TestCapitalizeString(t *testing.T) {
```

Comment:
For naming consistency this should be `TestStringCapitalization` or `TestCapitalizingString`. I think the later one is more easy to write and understand. Same rule applies to other functions.